backend/pom.xml

```diff
@@ -89,7 +89,7 @@
         <dependency>
             <groupId>javax.mail</groupId>
             <artifactId>mail</artifactId>
-            <version>1.4</version>
+            <version>1.5.0-b01</version>
```

Comment:
If `-b01` designates a beta version, I'd suggest going to the closest stable version